src/subscription_manager/cli_command/list.py

```diff
@@ -269,284 +54,38 @@ def __init__(self):
                 "and the server version (case-insensitive)."
```

Comment:
I think it would good to update this help text as well to more closely match the changed text in the man page for the `--matches` option to something like below:

> lists only installed products containing the specified expression in the product information, varying with the list requested and the server version (case-sensitive)